frontend/src/views/v-authorship.vue

```diff
@@ -293,6 +312,7 @@ export default {
       if (!files) {
         files = await window.api.loadAuthorship(this.info.repo);
       }
+      this.getRepoProps(repo);
```

Comment:
Can I ask about why this was moved?

Reply:
`getRepoProps` now calculates file type contributions with the actual files (instead of getting them from author) to filter out ignored files. Since files are only loaded in line 313, `getRepoProps` needs to be done after it.